src/test/java/be/syntra/devshop/DevshopBack/services/ProductServiceTest.java

```diff
@@ -268,4 +272,59 @@ void canFindProductRoundedPriceMaxTest(boolean archived) {
         assertThat(roundedResult).isEqualTo(price);
         verify(productRepository, times(1)).findRoundedMaxPrice(archived);
     }
+
+    @Test
+    void canSubmitReviewTest() {
+        // given
+        Product dummyProduct = createNonArchivedProduct();
+        final Review dummyReview = getDummyReview();
```

Comment:
same remark as the controllerTests. Create method in ReviewUtils, OR add to dummyProduct

Reply:
moved to getReviewSetWithDummyOtherReview() in ReviewUtils